### DISCUSSION TITLE
Add GH link to docs

### DISCUSSION THREAD

**Discussion by edalzell** [statamic-rad-pack/runway#325] [Feature Requests]
I was reading your (wonderful) docs, and I wanted to enter some issues/suggestions, but there was no link in the header of the docs to your repo.

Please add one so I don't have to remember what repo it's in?

**Comment by duncanmcclean** [comment]
Hah, fair enough! I'll see if I can get it added this week.

**Reply by duncanmcclean** [reply]
Added!
